doc/radosgw/multisite.rst

```diff
@@ -617,7 +617,7 @@ Failover and Disaster Recovery
 If the master zone should fail, failover to the secondary zone for
 disaster recovery.
 
-1. Make the secondary zone the master and default zone. For example:
+#. Make the secondary zone the master and default zone. For example:
```

Comment:
Yes, it's about document style consistency, either we fix it now, or fix when add/remove sections.

So let it be for now?